tests/integration/states/test_pkg.py

```diff
@@ -488,13 +489,17 @@ def test_pkg_009_latest_with_epoch(self):
         if not pkgmgr_avail(self.run_function, self.run_function('grains.items')):
             self.skipTest('Package manager is not available')
 
+        if (grains.get('os') == 'Amazon' and grains.get('osmajorrelease') != 2):
```

Comment:
This doesn't seem future proof. Maybe Amazon Linux 1 will support it in the future.

What if we test against a different package, that's available everywhere, like vim or something?

Reply:
Amazon Linux 1 reaches EOL June 20, 2020.  I think it is a done OS at this point.

Reply:
maybe to make it more future proof for all OSs, add a check to ensure the package exists, if it does not use bash-doc? what do you think about that approach @rares-pop